tests/suites/test_suite_entropy.function

```diff
@@ -447,7 +447,7 @@ void entropy_nv_seed_std_io()
 }
 /* END_CASE */
 
-/* BEGIN_CASE depends_on:MBEDTLS_MD_C:MBEDTLS_ENTROPY_NV_SEED:MBEDTLS_PLATFORM_NV_SEED_ALT */
+/* BEGIN_CASE depends_on:MBEDTLS_MD_LIGHT:MBEDTLS_ENTROPY_NV_SEED:MBEDTLS_PLATFORM_NV_SEED_ALT */
```

Comment:
What happens if `defined(MBEDTLS_MD_LIGHT) && !defined(MBEDTLS_MD_C)` ? Do we want the `entropy_nv_seed` test case to be run?

Reply:
It seems to me that both `entropy_nv_seed()` in `test_suite_entropy` and functions from `entropy.c` depend only on `MD_LIGHT` functions so this change should be fine. Am I missing something?